docs/sql-ref-syntax-ddl-repair-table.md

```diff
@@ -37,6 +37,13 @@ MSCK REPAIR TABLE table_identifier
 
     **Syntax:** `[ database_name. ] table_name`
 
+* **`[[ADD|DROP|SYNC] PARTITIONS]`**
```

Comment:
Is the syntax and function aiming to be identical with `HIVE-17824`?

Reply:
the syntax is from to the SQL standard but functionality is from Hive.